tools/sphere.c

```diff
@@ -0,0 +1,108 @@
+// Mystical Sphere of Slooping
+// Gon
+// 2017-04
+
+inherit NewThingCode;
+inherit AnsiTellCode;
+inherit "/usr/gon/tools/sloop";
+
+#include <ansi.h>
+
+// Functions
+int do_sloop( string arg );
+int do_touch( string str );
+int do_gaze( string str );
+
+void extra_create() {
+    set_short( "A Glowing Sphere" );
+    set_ids( ({"a glowing sphere", "glowing sphere", "sphere", 
+        "a sphere" }) );
+    set_long( "The sphere is glowing a deep purple that "
+        "seems to pulse rhythmically, almost as if the object "
+        "was breathing. The surface appears to be smooth "
```

Comment:
some of these lines are shorter than they need to be if you're formatting for 80 chars